idaes/gas_solid_contactors/unit_models/tests/test_MB.py

```diff
@@ -247,6 +251,10 @@ def test_solution(self, iron_oc):
                 iron_oc.fs.unit.gas_inlet.pressure[0] -
                 iron_oc.fs.unit.gas_outlet.pressure[0]) >= 0
```

Comment:
This check looks a little odd - it isn't actually checking for an expected result, just that pressure decreased over the column,

Reply:
I believe it may be redundant since the outlet pressure is asserted to be a lower value than the fixed inlet pressure. @chineduobiora do you believe we should keep this check?

Reply:
We should make this a check for an explicit value.

Reply:
@bpaul4 Yes, we should keep it, but update it to check for an explicit value (i.e. the outlet pressure) like @andrewlee94 mentioned.